pkg/network/tracer/connection/tracer.go

```diff
@@ -74,8 +74,37 @@ type Tracer interface {
 const (
 	defaultClosedChannelSize = 500
 	ProbeUID                 = "net"
+	connTracerModuleName     = "network_tracer_ebpf"
 )
 
+var ConnTracerTelemetry = struct {
+	tcpConns4         telemetry.Gauge
+	udpConns4         telemetry.Gauge
+	tcpConns6         telemetry.Gauge
+	udpConns6         telemetry.Gauge
+	tcpFailedConnects telemetry.Gauge
+	TcpSentMiscounts  nettelemetry.StatGaugeWrapper
+	missedTcpClose    telemetry.Gauge
+	missedUdpClose    telemetry.Gauge
+	UdpSendsProcessed nettelemetry.StatGaugeWrapper
+	UdpSendsMissed    nettelemetry.StatGaugeWrapper
+	UdpDroppedConns   telemetry.Gauge
+	PidCollisions     nettelemetry.StatGaugeWrapper
+}{
+	telemetry.NewGauge(connTracerModuleName, "tcp_conns4", []string{}, "Gauge measuring the number of active TCP4 connections in the EBPF map"),
+	telemetry.NewGauge(connTracerModuleName, "udp_conns4", []string{}, "Gauge measuring the number of active UDP4 connections in the EBPF map"),
+	telemetry.NewGauge(connTracerModuleName, "tcp_conns6", []string{}, "Gauge measuring the number of active TCP6 connections in the EBPF map"),
+	telemetry.NewGauge(connTracerModuleName, "udp_conns6", []string{}, "Gauge measuring the number of active UDP6 connections in the EBPF map"),
```

Comment:
Do we want separate metrics here, or could we use tags of `ip_proto:tcp`/`ip_proto:udp` and `family:v4`/`family:v6` for several of these stats.

Reply:
added